src/SprykerEco/Client/Computop/ComputopQuoteDefaultShipmentExpander.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Client\Computop;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+
+class ComputopQuoteDefaultShipmentExpander implements ComputopQuoteDefaultShipmentExpanderInterface
+{
+    /**
+     * @var \SprykerEco\Client\Computop\ComputopClientInterface
+     */
+    protected $computopClient;
```

Comment:
model should not use client from the same module
please inject respective model instaed